package.json

```
@@ -52,8 +52,8 @@
     "@process-engine/ci_tools": "2.2.0-beta.1",
     "@process-engine/dynamic_ui_core": "^1.5.0",
     "@process-engine/electron-updater": "1.0.0",
-    "@process-engine/management_api_client": "6.1.0-beta.3",
-    "@process-engine/process_engine_runtime": "9.0.0-beta.5",
+    "@process-engine/management_api_client": "6.2.0-alpha.2",
```

Die Betas für die Backendpakete wurden auch aktualisiert.

Hier kannst du jetzt diese Version verwenden: 

```suggestion
    "@process-engine/management_api_client": "6.1.0-beta.4",
```

Falls du dich wunderst, 6.1.0 ist hier richtig. 
Der Bump nach 6.2.0 auf dem Alpha channel ist ja nur passiert, weil ich davon ausging, dass die zuletzt gemergten Features nicht mehr in dieses Release kommen.
Da sie das aber nun doch tun, ist der Bump nach 6.2.0 nicht mehr notwendig.